package.json

```
@@ -245,8 +245,8 @@
     "@grafana/data": "workspace:*",
     "@grafana/e2e-selectors": "workspace:*",
     "@grafana/experimental": "1.7.10",
-    "@grafana/faro-core": "^1.3.6",
-    "@grafana/faro-web-sdk": "^1.3.6",
+    "@grafana/faro-core": "^1.4.2",
+    "@grafana/faro-web-sdk": "^1.4.2",
```

```suggestion
    "@grafana/faro-core": "1.4.2",
    "@grafana/faro-web-sdk": "1.4.2",
```
This should apply to the others too, I think

---

Although this should already cover minor releases. Do we want `^` or `~` just because we develop this internally?